db/migrations/mysql/000031_create_sidebar_categories.up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE IF NOT EXISTS SidebarCategories (
+    Id varchar(128) NOT NULL,
+    UserId varchar(26) DEFAULT NULL,
+    TeamId varchar(26) DEFAULT NULL,
+    SortOrder bigint(20) DEFAULT NULL,
+    Sorting varchar(64) DEFAULT NULL,
+    Type varchar(64) DEFAULT NULL,
+    DisplayName varchar(64) DEFAULT NULL,
+    Muted tinyint(1) DEFAULT 0,
+    Collapsed tinyint(1) DEFAULT 0,
```

Comment:
Currently we are not setting any default on initial creation. I am not against it but want to make sure we actually want it and if so we should be consistent and do it both for the new creation and migration cases.

Reply:
Hmm, good catch! These needs to be altered after initial creation 👍